truffle/scripts/testrpc-run.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+cd truffle
+# prepare testrpc accounts parameter string e.g. --account="0x11c..,1000" --account="0xc5d...,1000" ....
+source ./scripts/testrpc-accounts.sh
+
+# create variables
+configFile="../kin-sdk-core/src/androidTest/assets/testConfig.json"
```

Comment:
in future - would be nice to automatically generate this file from the env vars created by testrpc-accounts.sh

Reply:
this file also contains  token_contract_address, and it's not generated from testrpc-accounts.sh
so I don't know if it nicer.

Reply:
we have the test account private keys in 2 separate files. This is hard to maintain - but assuming we don't expect these to change, we can leave as is for now.